docs/release-guide.md

```diff
@@ -3,6 +3,43 @@ id: release-guide
 title: Release Guide
 ---
 
+## Release Flow
+
+The release flow of Apache APISIX follows [GitLab flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html), instead of Git flow or Github flow. Furthermore, [Release branches with GitLab flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html#release-branches-with-gitlab-flow) is the preferred way. The chart below will take the release `v2.3` of [apache/apisix-dashboard](https://github.com/apache/apisix-dashboard) as an example to illustrate the details.
+
+![Release Flow](/img/release_flow.png)
+
+The entire flow is comprised of the following four phases.
+
+### Planning Phase
+
+This phase will decide if a feature is ready to be released as well as the release time.
+
+- A new target milestone will be created in Github. (e.g. [`2.3`](https://github.com/apache/apisix-dashboard/milestone/6))
+- A discussion will be started on devel mailing list [dev@apisix.apache.org](dev@apisix.apache.org) for gathering ideas for the next release
```

Comment:
`devel`?

Reply:
@juzhiyuan use `dev mailing list` instead of `devel mailing list` ?

Reply:
yep

Reply:
OK, will renew soon.